app/views/shared/_cohort_charts.html.erb

```diff
@@ -32,7 +32,7 @@
         report_period = quarter_string(report_start)
         cohort_period = quarter_string(cohort_start)
       else
-        report_period = report_start.strftime("%b %Y")
+        report_period = [report_start.strftime("%b"), (report_start + 1.month).strftime("%b")].join(" - ")
```

Comment:
This labeling can be confusing. For example, does `Jul-Aug` represented
- (start of july) to (start of august) OR
- (start of july) to (end of august) 

Also if I want to look at what happened in month in a quarter, The labeling suggest there are only two bars to look at, but there should be 3.